ui/apps/everest/.e2e/utils/generic.ts

```diff
@@ -0,0 +1,23 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { expect } from '@playwright/test';
+
+export const checkError = async (response) => {
+  if (!response.ok()) {
+    console.log(`${response.url()}: `, await response.json());
+  }
+  expect(response.ok()).toBeTruthy();
```

Comment:
if we do `expect(response.ok()).toBeTruthy();` first, I think we can then just log the response, as so:

```
    expect(response.ok()).toBeTruthy();
    console.log(`${response.url()}: `, await response.json());
```

The way it is now, I think `expect(response.ok()).toBeTruthy()` is redundant, as you already checked if it was falsy in the first if clause.

Reply:
Fair point, haven't thought of that!